enterprise/models.py

```diff
@@ -2,42 +2,114 @@
 """
 Database models for enterprise.
 """
-
 from __future__ import absolute_import, unicode_literals
 
-# from django.db import models
+from autoslug import AutoSlugField
+from simple_history.models import HistoricalRecords  # likely a bug in import order checker
```

Comment:
@clintonb yes, if I sort them manually `isort --check-only` reports quality violation. If I run `isort` allowing to sort imports it places them in this order (and also adds some newlines).